### PR TITLE
added fullstacksherpa folder in participants folder

### LINKED ISSUE #13: [Task Assignment]: D&D 5e API Implementation
### Task Description
Implement a web project using HTML, CSS, and JavaScript that integrates a D&D 5e API.

### Task Requirements

- Create a new directory inside "participants" with your name.
- Develop necessary HTML, CSS, and JavaScript files.
- Integrate a D&D 5e API into your project.
- Update bootcamp.html with your name under `<navigation>`

### Submission Guidelines

- Create a branch: git checkout -b feature/your-feature-name
- Commit changes: git commit -m "Description of changes"
- Push to branch: git push origin feature/your-feature-name
- Create a Pull Request to merge changes into main.

### Additional Guidelines

- Maintain code quality and organization.
- Implement error handling and thorough testing.
- Document external APIs used.

### Resources
[D&D 5e API Documentation](https://www.dnd5eapi.co/)

### PR DESCRIPTION
**Pull request by fullstacksherpa**
### Description
-integrate GET from API[ https://www.dnd5eapi.co/docs/](https://www.dnd5eapi.co/docs/), using https://developer.mozilla.org/en-US/docs/Web/HTTP

### Changes Made
-added fullstacksherpa folder in participants folder and added fullstacksherpa list item in bootcamp.html

### Related Issues
[issue #13](https://github.com/org-45/yantra-lite/issues/13)
Resolves #13

### Checklist

- [x] I have tested these changes locally.
- [x] I have added/updated documentation as needed.
- [x] I have reviewed my code for potential issues.
- [x] I have followed the project's coding guidelines.
- [x] I have updated relevant comments and commit messages.
### Additional Information
-all the style are done using tailwindcss framwork.
-fetching data from D&D5eAPI

### By submitting this PR, I confirm that

- [x] I have read and understood the project's contributing guidelines.
- [x] This PR is ready for review and merging.
- [x] I understand that automated tests should pass before merging.

### CODE REVIEW

File: participants/fullstacksherpa/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function getData() {
+  const selectedCondition = document.getElementById("parameterSelect").value;
+  const apiUrl = `https://www.dnd5eapi.co/api/conditions/${selectedCondition}`;
+
+  // Make a GET request using fetch
+  fetch(apiUrl)
```
Comment (krishnemd) [inline participants/fullstacksherpa/index.js]: Can you try it with async await ?

File: participants/fullstacksherpa/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        throw new Error("Network response was not ok");
+      }
+      return response.json();
+    })
+    .then((data) => {
+      const resultElement = document.getElementById("result");
+      const descPoints = data.desc.map((point) => `<li>${point}</li>`).join("");
+
+      resultElement.innerHTML = `
+        <h3 class="text-center text-3xl">Description for the <span class="font-bold leading-4 tracking-widest text-purple-600">${selectedCondition} </span>condition</h3>
```
Comment (krishnemd) [inline participants/fullstacksherpa/index.js]: Do you have any alternative solution here? Instead of innerHTML method ?

**Review by krishnemd — CHANGES_REQUESTED** [review]

File: participants/fullstacksherpa/index.js (hunk 1 above)
Comment (fullstacksherpa) [reply participants/fullstacksherpa/index.js]: Sure,I will.

File: participants/fullstacksherpa/index.js (hunk 2 above)
Comment (fullstacksherpa) [reply participants/fullstacksherpa/index.js]: Is it okay to use document.createElement and appendChild instead of using innerHTML??

**Review by pfieffer — CHANGES_REQUESTED** [review]
As @chauchausoup suggested instead of the callbacks maybe using async await will be a better approach. 
But looks good. 
Thanks for the pr @fullstacksherpa

**Review by manjilkoirala — COMMENTED** [review]
Overall code seems good and working but as @chauchausoup  suggested use async and wait as you are calling from some REST API.
Also, you can try not to add HTML syntax inside JS.
Thank you for the PR☺️

**Review by pfieffer — APPROVED** [review]

**Pull request closed without merging**